Makefile

```diff
@@ -54,10 +56,23 @@ warmup: ## Warmup the cache
 purge: ## Purge cache and logs
 	rm -rf var/cache/* var/logs/*
 
+## —— Doctrine ▶️ ———————————————————————————————————————————————————————————————
+dgm: ## Generate a blank migration class
+	$(SYMFONY) doctrine:migrations:generate
```

Comment:
```suggestion
	$(SYMFONY) d:m:g
```

Reply:
I wanted to be explicit on Makefile. Shorter command is nice when you must to type it.

Reply:
In this case don't use a short code for the command clear cache

Reply:
You're right, i had change clear cache command to be explicit too.